cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-manager-rb/clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: curator-manager-rolebinding
```

Comment:
For each role/rolebinding can you include operator in the name?
ie: 
name: curator-operator-manager-rolebinding

Reply:
okay.